### DISCUSSION TITLE
why were configuration in ESP32 usually loose. how to fix it in code in Tasmota 13.0

### DISCUSSION THREAD

**Discussion by Rojer7084** [arendst/Tasmota#21531] [Support request]
I use tasmota 13.0 in ESP32  , when I configuration finish and I run some days  , the tasmota  will be reset and go to default configuration.   Anyone found the problem like me?  How to solved it.  Thanks for your help.

**Comment by sfromis** [comment]
As you have not analyzed your issue, or the circumstances, not possible to tell what the problem is, or how to solve. Might be power fluctuations, or buttons registering as "pushed" for 40 seconds or more. Or something....
